src/TrainingNet.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App"/>
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.1.1.1"/>
+  </ItemGroup>
+</Project>
```

Comment:
Add an empty line at EOF

Reply:
This files get updated when adding new libraries, so make sure you check them out when using the NuGet Package Manager